c/tskit/tables.c

```diff
@@ -8421,7 +8453,8 @@ tsk_table_collection_add_and_remap_node(tsk_table_collection_t *self,
             population_map[node.population] = node.population;
         }
         if (population_map[node.population] == TSK_NULL) {
-            tsk_population_table_get_row(&other->populations, node.population, &pop);
+            tsk_population_table_get_row_unsafe(
+                &other->populations, node.population, &pop);
```

Comment:
Same here, how do we know `other->nodes` has valid individuals and populations? I followed it back a bit but wasn't obvious to me.